### PR TITLE
Fixes #2943 Add the property filter to tooltip on Pie chart

### LINKED ISSUE #2943: Add the property filter to tooltip on Pie chart
## Is your feature request related to a problem?

When looking at a normal graph, you can distinguish which line is filtered on what:
image

But in the Pie chart, we don't show that info:
image

## Describe the solution you'd like
Show the filtering information on the pie chart, like the line graph.

### PR DESCRIPTION
**Pull request by c3ho**
## Changes

*Please describe.*  
*If this affects the frontend, include screenshots.*  
Fixes #2943 

Added filters to pie chart labels. Hovering over the pie chart will now display the relevant filters used. I also made the look of the labels to look like the ones from the line graph.

I apologize for the small after image...

Before:
image

After:
image

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests

### CODE REVIEW

**Comment by Twixes** [comment]
Thanks for the contribution @c3ho and sorry for the delay! I refactored the code slightly, I'll hand it off for review to a teammate.

**Comment by c3ho** [comment]
@Twixes no worries!

File: frontend/src/scenes/insights/LineGraph.js [hunk 1]
```diff
@@ -21,6 +21,16 @@ Chart.defaults.global.elements.line.tension = 0
 
 const noop = () => {}
 
+const commonTooltipsOptions = {
+    enabled: true,
+    yPadding: 10,
+    xPadding: 10,
+    caretPadding: 0,
+    backgroundColor: '#1dc9b7',
+    titleFontColor: '#ffffff',
```
Comment (paolodamico) [inline frontend/src/scenes/insights/LineGraph.js:30]: Tried to change this to have colors that have strong contrast ratio. Current scheme doesn't work very well, but `titleFontColor` doesn't actually change the color of the label.

File: package.json [hunk 2]
```diff
@@ -148,7 +148,7 @@
     },
     "lint-staged": {
         "*.{js,ts,tsx,json,yml,css,scss}": "prettier --write",
-        "*.{js,ts,tsx}": "eslint",
+        "*.{js,ts,tsx}": "eslint --fix",
```
Comment (paolodamico) [inline package.json:151]: Not 100% sure about this? Could this potentially lead to some unexpected changes? I do think that most of the time the fixes are quite straightforward

**Review by paolodamico — APPROVED** [review]
Couple of partially unrelated points, but lgtm!

**Comment by Twixes** [comment]
Could you take over from here @samwinslow given involvement with graphs? Would be cool to merge this!

**Comment by Twixes** [comment]
Hi! We've reworked our graphs and tooltips recently, so there's a conflict here that's hard to resolve. And we'll be addressing this with our new tooltips too. They look pretty good – see below:

Tip

I'll close this, but still, thank you very much for the contribution.

**Pull request closed without merging**